src/main/java/org/jvnet/hudson/test/JenkinsHomeLoader.java

```diff
@@ -36,11 +36,11 @@
 import javax.annotation.CheckForNull;
 
 /**
- * Controls how a {@link HudsonTestCase} initializes <tt>JENKINS_HOME</tt>.
+ * Controls how a {@link JenkinsTestCase} initializes <tt>JENKINS_HOME</tt>.
```

Comment:
Since HudsonTestCase is deprecated, it should not be renamed.

```suggestion
 * Controls how a {@link HudsonTestCase} initializes <tt>JENKINS_HOME</tt>.
```